src/utils/is-metadata.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Checking if and key is metadata.
```

Comment:
Consider rewording: `Checking if key is metadata`

Reply:
Fixed: Checking if key is metadata